### ISSUE TITLE
Auth with webview throws AttributeError: can't set attribute

### ISSUE THREAD

**Issue by Column01** [derrod/legendary#433]
## Platform

Operating system and version: Windows 10 64 bit, version 19044.1706
Legendary version (`legendary -V`): `legendary version "0.20.25", codename "Our Benefactors"`

## Expected Behavior

When running the auth command, it should open the webvite

## Current Behavior

Program crashes with an `AttributeError`

## Steps to Reproduce

1. Install legendary with the webview mode
2. Run `legendary auth`
3. Program crashes

## Additional information

The error goes away when using `--disable-webview` obviously. Here's the traceback for the crash:
```
PS C:\WINDOWS\system32> legendary auth
[cli] INFO: Testing existing login data if present...
[LGDLFS] DEBUG: Failed to load user data: FileNotFoundError(2, 'No such file or directory')
[WebViewHelper] INFO: Opening Epic Games login window...
Traceback (most recent call last):
  File "c:\users\colin\appdata\local\programs\python\python38\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\colin\appdata\local\programs\python\python38\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\Colin\AppData\Local\Programs\Python\Python38\Scripts\legendary.exe\__main__.py", line 7, in <module>
  File "c:\users\colin\appdata\local\programs\python\python38\lib\site-packages\legendary\cli.py", line 2913, in main
    cli.auth(args)
  File "c:\users\colin\appdata\local\programs\python\python38\lib\site-packages\legendary\cli.py", line 168, in auth
    if do_webview_login(callback_sid=self.core.auth_sid, callback_code=self.core.auth_code):
  File "c:\users\colin\appdata\local\programs\python\python38\lib\site-packages\legendary\utils\webview_login.py", line 134, in do_webview_login
    window.loaded += api.on_loaded
AttributeError: can't set attribute
```

**Comment by CommandMC** [comment]
Fixed with ec91f69adc0fab6ee4d3d5e7864237b53fb66df2

**Comment by derrod** [comment]
This was fixed with #423, although a new release has not yet been made. For the time being you can downgrade pywebview (although I don't know the exact version that broke it).

**Comment by Column01** [comment]
Thanks for the info, I'll just implement the fix locally until a new release can be made!
